lib/dirac.js

```diff
@@ -471,6 +471,10 @@ dirac.tx = {
       tx[ tbl ] = new Dal( client );
       tx[ tbl ].befores = dirac.dals[ tbl ].befores;
       tx[ tbl ].afters  = dirac.dals[ tbl ].afters;
+      tx[ tbl ].lock = function(mode, callback) {
```

Comment:
Maybe - but that'd basically be doing what JS is already doing for us - scoping a function call to a specific object.

The only overhead TX creates is the new contexts in memory for each DAL, all of which hold a reference to the new client. I think adding a `client` option just changes which object we're getting the client from.